priv/posts/marketing/20240723023712_getting-to-know-adolfo-neto.md

```diff
@@ -0,0 +1,95 @@
+
+We have a modest audience for Brazil, around 200 people download our episodes every month. We publish one episode each month, we meet new people, and we learn a lot by doing it. I see only advantages in doing that podcast.
+
+#### How has the Erlang Ecosystem Foundation supported you in this project?
+
+The Erlang Ecosystem Foundation supports Elixir em Foco Podcast. Thanks to their support we can use a great software for recording our interviews. It makes the editing process a lot easier.
+
+Additionally, I am part of the [Hello Erlang podcast](https://helloerlang.github.io/), created by Slava (Viacheslav Katsuba), a Ukrainian Erlanger. The podcast is currently on pause due to the Russian invasion of Ukraine, but we hope to return recording interviews soon. The Foundation is also going to support this podcast.
+
+#### Can you share details about the expanding Elixir community in Brazil?
+
+It's a growing community. I believe many people try to learn Elixir for the same reason that I did, because it was created by a Brazilian. José Valim is a great guy, he is always helping. I've had students who had questions and reached out to Valim; he always responds.
+
+And also, there are some companies in Brazil that use Elixir in production. But one problem is that the community has grown so much that there are not enough jobs in Brazil for the people that want to work with Elixir. We need more companies adopting Elixir, we need more citizens learning what you can do with Elixir in Erlang, in the BEAM. 
+
+As a professor, I make an effort to introduce my students to Elixir and Erlang languages because there isn't a dedicated course only focused on learning Elixir. Without this effort, some students may graduate without ever discovering Elixir, a programming language created by a Brazilian that's widely adopted by both Brazilian and international companies.
+
+#### What excites you about the future of Erlang/OTP, Elixir, etc.? Is there anything you are particularly looking forward to?
+
+I'm excited about exploring the new machine learning features available in Livebook, Nx, and Scholar. At the Elixir Days event, Paulo Valente from the Nx core team gave an insightful talk. He talked about many interesting things that you can do with machine learning in Elixir using Livebook, which is another great project. One area I haven't had the chance to delve into yet is Nerves, mainly because I'm not much of a hardware guy. To experiment with Nerves, I'd need some embedded hardware like Arduino or similar devices.
+
+And also we want to, Lee Barney is working on that, create resources for people who don't know how to program, so they can learn Elixir using a resource, as a website. That's what I want to see in the future, it would be great. Of course, we have Elixir school which is great, but its focus is more on people that already know a programming language.

```

Comment:
```suggestion
And also we want to create resources such as a website for people who don't know how to program, so they can learn Elixir using a resource, Lee Barney is working on that. That's what I want to see in the future, it would be great. Of course, we have Elixir school which is great, but its focus is more on people that already know a programming language.

```